monkey/infection_monkey/control.py

```diff
@@ -113,14 +114,32 @@ def find_server(default_tunnel=None):
                 logger.info("Starting tunnel lookup...")
                 proxy_find = tunnel.find_tunnel(default=default_tunnel)
                 if proxy_find:
-                    proxy_address, proxy_port = proxy_find
-                    logger.info("Found tunnel at %s:%s" % (proxy_address, proxy_port))
-                    ControlClient.proxies["https"] = PROXY_SCHEMA % (proxy_address, proxy_port)
+                    ControlClient.set_proxies(proxy_find)
                     return ControlClient.find_server()
                 else:
                     logger.info("No tunnel found")
                     return False
 
+    @staticmethod
+    def set_proxies(proxy_find):
+        """
+        Note: Proxy schema changes! When upgrading to newer python version or
+        when urllib3 and requests are updated there is possibility that the proxy
+        schema is changed.
+        https://github.com/psf/requests/issues/5297
+        https://github.com/psf/requests/issues/5855
+        """
+        proxy_address, proxy_port = proxy_find
+        logger.info("Found tunnel at %s:%s" % (proxy_address, proxy_port))
+        if is_windows_os():
+            ControlClient.proxies["https"] = "http://%s:%s" % (
+                proxy_address,
+                proxy_port,
+            )
+        else:
+            ControlClient.proxies["https"] = PROXY_SCHEMA % (proxy_address, proxy_port)
```

Comment:
Lines 135 and 140 are inconsistent. Either use a constant for both or inline the string for both. I would just advocate for using format strings in both cases (e.g. `f"http://{proxy_address}:{proxy_port}"`)